pkgs/os-specific/linux/kernel/generic.nix

```diff
@@ -1,13 +1,11 @@
-{ buildPackages, runCommand, nettools, bc, perl, gmp, libmpc, mpfr, openssl
+{ buildPackages
 , ncurses
-, libelf
-, utillinux
-, writeTextFile, ubootTools
 , callPackage
 , overrideCC, gcc7
+, perl
 }:
 
-{ stdenv, buildPackages, perl, buildLinux
```

Comment:
I moved the perl into the other set because I don't think people override it and if so it makes more sense in the callPackage call.